bigtable/examples/run_examples_utils.sh

```diff
@@ -45,7 +45,6 @@ function run_all_instance_admin_examples {
   echo
   echo "Run create-instance example."
   $admin ../examples/bigtable_samples_instance_admin create-instance "${project_id}" "${INSTANCE}" "${zone_id}"
-  trap '$admin ../examples/bigtable_samples_instance_admin delete-instance "${project_id}" "${INSTANCE}"' EXIT
```

Comment:
This seems unrelated to the description of this PR, why the change?

Reply:
I should probably separate them to a different PR, they were just breaking my tests because (see next comment)..

Reply:
SGTM, if you don't mind, that would make sense.